addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -139,6 +139,14 @@ $-editor-messages-margin-x: 2%;
 [draggable] {
     user-select: none;
 }
+/* Prevent the native drag and drop for images. */
+img {
+    -webkit-user-drag: none !important;
+    -khtml-user-drag: none !important;
+    -moz-user-drag: none !important;
+    -o-user-drag: none !important;
+    user-drag: none !important;
+}
```

Comment:
The other alternative would be to use [the draggable attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable) but it would require a lot more changes... (add attribute on edition start or upgrade script, change every image template or have an observer that add the attribute on every images...)

Also, maybe this commit can be backported if we want it in stable